dsaps/cli.py

```diff
@@ -76,14 +81,16 @@ def main(ctx, config_file, url, email, password):
         handlers=[logging.FileHandler(f"logs/log-{log_suffix}", "w")],
         level=logging.INFO,
     )
-    logger.info("Application start")
-    client = Client(url)
-    client.authenticate(email, password)
-    start_time = time.time()
-    ctx.obj["config"] = helpers.load_source_config(config_file)
-    ctx.obj["client"] = client
-    ctx.obj["start_time"] = start_time
-    ctx.obj["log_suffix"] = log_suffix
+    logger.info("Running process")
+    source_config = helpers.load_source_config(config_file)
+    if url:
+        dspace_client = dspace.DSpaceClient(url)
+        dspace_client.authenticate(email, password)
+        ctx.obj["dspace_client"] = dspace_client
+    ctx.obj["config"] = source_config
```

Comment:
Maybe rename `config` attribute to `source_config` for consistency and to minimize potential confusion?